ansible/roles/jenkins-worker/vars/main.yml

```diff
@@ -97,7 +97,8 @@ java_path: {
   'macos10.14': 'java',
   'macos10.15': 'java',
   'macos10.16': 'java',
-  'macos11.0': 'java',
+  # Currently hardcoded untill adopt have their build available
+  'macos11.0': '/Users/administrator/zulu8.52.0.23-ca-jdk8.0.282-macosx_aarch64/bin/java',
```

Comment:
Is this a "fat" binary that would run on both intel and arm?

Reply:
no this is an arm specific binary because the using an intel one caused the jenkins agent to launch in x64 mode and wouldnt allow for building of an arm binary for testing

Reply:
ok, not so worried about what we use for Java, but was wondering if they provided fat binaries that were usable as well.  I assume this will not prevent us from building/testing a fat binary, right?